src/ui/rendering/handlebarsrenderer.js

```diff
@@ -194,8 +194,24 @@ export default class HandlebarsRenderer extends Renderer {
     });
 
     this.registerHelper('processTranslation', function (options) {
+      const pluralizationInfo = {};
+      const interpolationParams = {};
       let { phrase, count } = options.hash;
-      return TranslationProcessor.process(phrase, options.hash, count);
+
+      Object.entries(options.hash).forEach(([key, value]) => {
+        if (key === 'locale') {
+          pluralizationInfo['locale'] = value;
+        } else if (key.length > 10 && key.substring(0, 10) === 'pluralForm') {
+          const pluralFormNum = parseInt(key.substring(10));
```

Comment:
Are there languages with more than 9 plural forms?

Reply:
According to [this](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals), the most any language has is 6.